app.yaml

```diff
@@ -11,10 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: custom
+runtime: ruby
 vm: true
 api_version: 1
 module: default
+# [START entrypoint]
+entrypoint: bundle exec foreman start --formation "$FORMATION"
+# [END entrypoint]
 
 resources:
```

Comment:
Apologies, you're right - entrypoint should be there too.  Node.js requires no entrypoint because of npm start :P